src/Pages/ProductDetail/Component/ProductDesc/ProductDesc.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import ProductFeatures from './ProductFeatures/ProductFeatures';
+import styled from 'styled-components';
+import theme from '../../../../style/Theme';
+import Rating from '@mui/material/Rating';
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
+
+const ProductDesc = ({ product }) => {
+  const {
+    name,
+    price,
+    // bold,
+    // tannic,
+    // sweet,
+    // acidic,
+    ratings,
+    paring,
+    winery,
+    type,
+    country,
+    grape,
+    // paring_image,
+    image,
+  } = product;
+
+  const productFeatures = [
+    { winery: winery },
+    { type: type },
+    { grape: grape },
+    { country: country },
```

Comment:
key value가 동일한 이름이라면 단축속성명을 활용할 수 있습니다
```suggestion
    { country},
```

Reply:
반영하였습니다. 앞으로도 활용하겠습니다. 감사합니다.